:memo: Clarify project level timezone settings (#2172)
platformsh/platformsh-docs: docs/src/configuration/app/timezone.md

{{< /codetabs >}}

Setting the application timezone only affects the application itself, not system operations such as log files.

> The timezone [can be set on a project level](../../administration/web/configure-project.md#general) but it won't impact the App runtime.

Reviewer: ```suggestion
You can also set the [timezone for a project](../../administration/web/configure-project.md#general), but it doesn't affect the timezone for the app runtime.
```

Partly to line up with Vale's recommendation. Partly to be more active.
